src/NumericalAlgorithms/DiscontinuousGalerkin/MortarHelpers.hpp

```diff
@@ -56,6 +57,12 @@ Variables<Tags> project_to_mortar(
     const Mesh<Dim>& face_mesh,
     const Mesh<Dim>& mortar_mesh,
     const std::array<Spectral::MortarSize, Dim>& mortar_size) noexcept {
+  if (face_mesh == mortar_mesh and
+      alg::all_of(mortar_size, [](const Spectral::MortarSize& size) noexcept {
+        return size == Spectral::MortarSize::Full;
+      })) {
+    return vars;
```

Comment:
Does this actually help?  This function already just calls `std::copy` on the raw data array in this case.  If that's slow we should fix it in `apply_matrices`, not work around it here.

Reply:
I checked and it was just being misattributed in the profile. At least at the overall simulation level I can't measure I time difference.